R/qza_to_tse.R

```diff
@@ -0,0 +1,107 @@
+#' Generates a TreeSummarizedExperiment (TSE) object from .qza artifacts
+#'\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'
+#' Construct a TreeSummarizedExperiment object from multiple 
+#' qiime2 artifacts (.qza). 
+#' '\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
```

Comment:
why this line is here?

Reply:
I accidentally moved it around when changing the code, should be good now